x-pack/plugins/security_solution/public/actions/filter/lens/create_action.ts

```diff
@@ -95,7 +98,24 @@ export const createFilterLensAction = ({
         ? services.timelineFilterManager
         : dataService.query.filterManager;
 
-      addFilter({ filterManager, fieldName: field, value, dataViewId });
+      // If value type is value_count, we want to filter an `Exists` filter instead of a `Term` filter
+      if (valueType === 'value_count') {
+        addFilter({
+          filterManager,
+          fieldName: field,
+          value: [],
+          negate: !!negate,
```

Comment:
NIT: maybe a couple of comments here would help understand:

```suggestion
          value: [], // set empty "value" to force adding an "exist" filter.
          negate: !!negate, // force the "negate" to avoid using the "value" to calculare it
```

Reply:
Resolving in favor of [this](https://github.com/elastic/kibana/pull/181151#discussion_r1576607256), we won't need these comments.